test/jackdaw/client/partitioning_test.clj

```diff
@@ -1,4 +1,4 @@
-(ns jackdaw.client-test
+(ns jackdaw.client.partitioning-test
```

Comment:
This is the correct ns name (unless you want to move the file).

Reply:
Ah thanks for fixing this.